XamarinCommunityToolkit.UnitTests/Behaviors/NumericValidationBehavior_Tests.cs

```diff
@@ -12,39 +13,62 @@ public NumericValidationBehavior_Tests()
 
 		[Theory]
 		// Positive
-		[InlineData("15.2", 1.0, 16.0, 0, 16, true)]
-		[InlineData("15.", 1.0, 16.0, 0, 1, true)]
-		[InlineData("15.88", 1.0, 16.0, 2, 2, true)]
-		[InlineData("0.99", 0.9, 2.0, 0, 16, true)]
-		[InlineData(".99", 0.9, 2.0, 0, 16, true)]
+		[InlineData("en-US", "15.2", 1.0, 16.0, 0, 16, true)]
+		[InlineData("en-US", "15.", 1.0, 16.0, 0, 1, true)]
+		[InlineData("en-US", "15.88", 1.0, 16.0, 2, 2, true)]
+		[InlineData("en-US", "0.99", 0.9, 2.0, 0, 16, true)]
+		[InlineData("en-US", ".99", 0.9, 2.0, 0, 16, true)]
+		[InlineData("de-DE", "15,2", 1.0, 16.0, 0, 16, true)]
+		[InlineData("de-DE", "15,", 1.0, 16.0, 0, 1, true)]
+		[InlineData("de-DE", "15,88", 1.0, 16.0, 2, 2, true)]
+		[InlineData("de-DE", "0,99", 0.9, 2.0, 0, 16, true)]
+		[InlineData("de-DE", ",99", 0.9, 2.0, 0, 16, true)]
 		// Negative
-		[InlineData("15.3", 16.0, 20.0, 0, 16, false)]
-		[InlineData("15.3", 0.0, 15.0, 0, 16, false)]
-		[InlineData("15.", 1.0, 16.0, 0, 0, false)]
-		[InlineData(".7", 0.0, 16.0, 0, 0, false)]
-		[InlineData("15", 1.0, 16.0, 1, 16, false)]
-		[InlineData("", 0.0, 16.0, 0, 16, false)]
-		[InlineData(" ", 0.0, 16.0, 0, 16, false)]
-		[InlineData(null, 0.0, 16.0, 0, 16, false)]
-		public void IsValid(string value, double minValue, double maxValue, int minDecimalPlaces, int maxDecimalPlaces, bool expectedValue)
+		[InlineData("en-US", "15.3", 16.0, 20.0, 0, 16, false)]
+		[InlineData("en-US", "15.3", 0.0, 15.0, 0, 16, false)]
+		[InlineData("en-US", "15.", 1.0, 16.0, 0, 0, false)]
+		[InlineData("en-US", ".7", 0.0, 16.0, 0, 0, false)]
+		[InlineData("en-US", "15", 1.0, 16.0, 1, 16, false)]
+		[InlineData("en-US", "", 0.0, 16.0, 0, 16, false)]
+		[InlineData("en-US", " ", 0.0, 16.0, 0, 16, false)]
+		[InlineData("en-US", null, 0.0, 16.0, 0, 16, false)]
+		[InlineData("de-DE", "15,3", 16.0, 20.0, 0, 16, false)]
+		[InlineData("de-DE", "15,3", 0.0, 15.0, 0, 16, false)]
+		[InlineData("de-DE", "15,", 1.0, 16.0, 0, 0, false)]
+		[InlineData("de-DE", ",7", 0.0, 16.0, 0, 0, false)]
+		[InlineData("de-DE", "15", 1.0, 16.0, 1, 16, false)]
+		[InlineData("de-DE", "", 0.0, 16.0, 0, 16, false)]
+		[InlineData("de-DE", " ", 0.0, 16.0, 0, 16, false)]
+		[InlineData("de-DE", null, 0.0, 16.0, 0, 16, false)]
```

Comment:
[InlineData("en-US", "15,2", 1.0, 16.0, 0, 16, false)]
[InlineData("de-DE", "15.2", 1.0, 16.0, 0, 16, false)]

I think we should add unit tests for testing negative scenarios (we pass valid en-US number format with de-DE culture and vice versa. I hope you got the idea)

Reply:
I added a few more tests. please let me know if it's enough.

Reply:
awesome